### PR TITLE
BD-2685: 404 fix

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**BD-2685**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_user_guide/personalization_and_dynamic_content/connected_content/aborting_connected_content.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ In the following example, the conditionals `connected.recommendations.size < 5`
    {% endif %}
 ```
 
-You can also specify an abort reason, which will be saved to the [Message Activity Log]({{site.baseurl}}/user_guide/administrative/app_settings/developer_console/message_activity_log_tab/). This abort reason must be a string and cannot contain Liquid.
+You can also specify an abort reason, which will be saved to the [Message Activity Log]({{site.baseurl}}/docs/user_guide/administrative/app_settings/message_activity_log_tab/). This abort reason must be a string and cannot contain Liquid.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/personalization_and_dynamic_content/connected_content/aborting_connected_content.md:23]:
```suggestion
You can also specify an abort reason, which will be saved to the [Message Activity Log]({{site.baseurl}}/user_guide/administrative/app_settings/message_activity_log_tab/). This abort reason must be a string and cannot contain Liquid.
```

File: _docs/_user_guide/personalization_and_dynamic_content/connected_content/aborting_connected_content.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ In the following example, the conditionals `connected.recommendations.size < 5`
    {% endif %}
 ```
 
-You can also specify an abort reason, which will be saved to the [Message Activity Log]({{site.baseurl}}/user_guide/administrative/app_settings/developer_console/message_activity_log_tab/). This abort reason must be a string and cannot contain Liquid.
+You can also specify an abort reason, which will be saved to the [Message Activity Log]({{site.baseurl}}/user_guide/administrative/app_settings/message_activity_log_tab/). This abort reason must be a string and cannot contain Liquid.
```
Comment (internetisaiah) [inline _docs/_user_guide/personalization_and_dynamic_content/connected_content/aborting_connected_content.md:23]: ## Summary

Hey something weird is going on here for me. When I go to [the page](https://braze-docs-q4jsld1nv-braze.vercel.app/docs/user_guide/personalization_and_dynamic_content/connected_content/aborting_connected_content) and test the new link, the link page goes to the following link and generates a 404:

`/docs/user_guide/personalization_and_dynamic_content/connected_content/%7B%7Bsite.baseurl%7D%7D/user_guide/administrative/app_settings/message_activity_log_tab`

Is this happening to you as well?

### Things I tried:

- Testing in your latest Vercel deployment as well as my local environment
- Testing on Chrome and Firebox, both signed-in and incognito.
- Added a space between `{{site.baseurl}}` and the first `/`.

Comment (internetisaiah) [reply _docs/_user_guide/personalization_and_dynamic_content/connected_content/aborting_connected_content.md:23]: Follow up. This is happening on the live site as well?

- [Page with link](https://www.braze.com/docs/user_guide/personalization_and_dynamic_content/connected_content/aborting_connected_content)
- [Page the link _tries_ to go to](https://www.braze.com/docs/user_guide/administrative/app_settings/message_activity_log_tab/?redirected=true)
- [Page the link actually goes to](https://www.braze.com/docs/user_guide/personalization_and_dynamic_content/connected_content/%7B%7Bsite.baseurl%7D%7D/user_guide/administrative/app_settings/developer_console/message_activity_log_tab)

Comment (bre-fitzgerald) [reply _docs/_user_guide/personalization_and_dynamic_content/connected_content/aborting_connected_content.md:23]: Thank you for the thorough QA 🙏 pushing a fix now

**Review by internetisaiah — APPROVED** [review]
LGTM

**Pull request merged**